docs/resources/service_level_objective.md

```diff
@@ -79,7 +79,7 @@ resource "datadog_service_level_objective" "bar" {
 
 - `name` (String) Name of Datadog service level objective
 - `thresholds` (Block List, Min: 1) A list of thresholds and targets that define the service level objectives from the provided SLIs. (see [below for nested schema](#nestedblock--thresholds))
-- `type` (String) The type of the service level objective. The mapping from these types to the types found in the Datadog Web UI can be found in the Datadog API [documentation page](https://docs.datadoghq.com/api/v1/service-level-objectives/#create-a-slo-object). Valid values are `metric`, `monitor`.
+- `type` (String) The type of the service level objective. The mapping from these types to the types found in the Datadog Web UI can be found in the Datadog API [documentation page](https://docs.datadoghq.com/api/v1/service-level-objectives/#create-a-slo-object). Valid values are `metric`, `monitor`, `time_slice`.
```

Comment:
```suggestion
- `type` (String) The type of the service level objective. The mapping from these types to the types found in the Datadog Web UI can be found in the [Datadog API documentation](https://docs.datadoghq.com/api/v1/service-level-objectives/#create-a-slo-object). Valid values are `metric`, `monitor`, `time_slice`.
```

Reply:
Thanks for the review! This part of the code isn't really owned by my team, this change was generated automatically following an update of the API client. I believe this PR introduced it: https://github.com/DataDog/datadog-api-client-go/pull/2328. Do you mind if I leave it like this?

Reply:
You bet. Gotcha, that makes sense. No worries--I think it's fine to leave then. 😄